### PR TITLE
Added Graph, added styling from b946b9876db2b7c983d252b711384e42b2091dc7

### PR DESCRIPTION
**Pull request by kris927b**
I added a graph element using plot.ly JS API. The graph plots both the validation loss and the validation accuracy. 
Moreover, I included the styling of the page as presented in wuwica's commit b946b9876db2b7c983d252b711384e42b2091dc7

### CODE REVIEW

**Comment by shiffman** [comment]
I just merged #2, would you mind updating this to work with that pull request? Sorry, I'm doing them in reverse order!

**Comment by kris927b** [comment]
Of course, I am on it!

**Comment by shiffman** [comment]
Thanks, there is some buggy stuff about the layout etc after the merge so feel free to fix whatever you like! If you can include a link to a demo or screenshot as part of the pull request that makes it easier to review too! Thank you 😄

**Comment by kris927b** [comment]
So I made the changes so that every thing should work. 
Moreover, here is a screen shot, of what it looks like at my end. :smile: 
graph

**Comment by kris927b** [comment]
I am sorry for the many small commits. 
I am still a bit new to the contribution side of Github. :confused: 
But now, everything should be as represented at the previous picture. :smile:

**Comment by shiffman** [comment]
Amazing! It's all good, the point of this repo is for learning and experimentation! 🚂 💯
